src/main/webapp/customer-center/subscriptions.jsp

```diff
@@ -0,0 +1,77 @@
+<%@ page import="hu.ak.generics.cafeloop.util.DeliveryUtils" %>
+<%@ page import="hu.ak.generics.cafeloop.util.DateUtils" %>
+<%@ page contentType="text/html; charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="sql" uri="http://java.sun.com/jsp/jstl/sql" %>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
+<html>
```

Comment:
Ez a JSP felel az előfizetések listázásáért. Nem túl szép az oldal, de úgy gondoltam, hogy meghagyom így, ahogy van, hátha homokozóként szolgálhat nektek abban, hogy kipróbáljátok ti is az oldal fejlesztgetését vagy legalábbis a megpróbálását.